src/test/components/TabBar.test.js

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+import React from 'react';
+import { render, fireEvent, cleanup } from 'react-testing-library';
+import TabBar from '../../components/app/TabBar';
+
+afterEach(cleanup);
+
+it('renders the TabBar', () => {
+  const myMock = jest.fn();
+  render(
+    <TabBar
+      selectedTabSlug="flame-graph"
+      visibleTabs={[
+        'calltree',
+        'flame-graph',
+        'stack-chart',
+        'marker-chart',
+        'marker-table',
+        'network-chart',
+      ]}
+      onSelectTab={myMock}
+    />
+  );
```

Comment:
In this test, you don't expect anything. So the test only tests that we can render the component, it doesn't test that the component renders properly.

To test that the component renders properly, we can use a snapshot-based test. This is how it goes:

```js
const { container } = render(...);
expect(container.firstChild).toMatchSnapshot();
```

Jest (our test runner) will automatically store the snapshot in a new file, and will check it everytime the test runs. This way we can check that the output doesn't change, and we can even check the output in a separate file. We use snapshot testing extensively in our tests because this is very easy to use and very effective to check problems and output changes.